docs/meshstack.metadata-tags.md

```diff
@@ -181,16 +181,25 @@ Every platform might have different limitations about the tag names and values,
-| `${paymentName}`           | The display name of the payment method that is assigned to the meshProject                                                                                     |
-| `${paymentExpirationDate}` | The expiration date of the payment method that is assigned to the meshProject. It contains `none` if no expiration date is set for the payment method.         |
+| Tag Key                       | Description                                                                                                                                                     |
+| ----------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `${projectIdentifier}`        | The project identifier                                                                                                                                          |
+| `${workspaceIdentifier}`      | The project's workspace identifier                                                                                                                              |
+| `${workspaceName}`            | The projects's workspace name                                                                                                                                   |
+| `${projectName}`              | The project display name                                                                                                                                        |
+| `${landingzone}`              | The name of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the project does not have a Landing Zone applied       |
+| `${landingzoneIdentifier}`    | The identifier of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the project does not have a Landing Zone applied |
+| `${paymentIdentifier}`        | The identifier of the payment method that is assigned to the project                                                                                            |
+| `${paymentName}`              | The display name of the payment method that is assigned to the project                                                                                          |
+| `${paymentExpirationDate}`    | The expiration date of the payment method that is assigned to the project. It contains `none` if no expiration date is set for the payment method               |
+| `${paymentAmount}`            | The amounf of the assigned payment method. It is `none` if no payment method is set                                                                             |
+| `${contactMail}`              | Contains the owners email for the workspace of this project. It is `none` if no owner is set                                                                    |
```

Comment:
```suggestion
| `${contactMail}`              | The email of the workspace owner of this project. It is `none` if no owner is set                                                                    |
```
To be consistent with the other descriptions.

Reply:
changed